root/admin/wikidoc/DeleteWikiDoc.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *     | o  ((*   l  /
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormSubmit from '../../components/FormSubmit';
+import Layout from '../../layout';
+import {withCatalystContext} from '../../context';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: FormT<{}>,
+  +page: string,
+};
+
+const DeleteWikiDoc = withCatalystContext(({
+  $c,
+  form,
+  page,
+}: Props) => (
+  <Layout fullWidth title={l('Remove Page')}>
+    <div id="content">
+      <h1>{l('Remove Page')}</h1>
+      <p>
+        {exp.l(`Are you sure you wish to remove
+                <strong>{page}</strong> from the transclusion table?`,
+               {page: page})}
```

Comment:
Not on the TT, but maybe we could actually link to the page while at it?

Reply:
Good idea, linked to it in https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/3f533283a2bf22bee7d32223607e096d3a926134.